src/components/container/ExternalAccountsPreview/ExternalAccountsPreview.tsx

```diff
@@ -1,65 +1,91 @@
-import React, { useState, useEffect } from 'react'
+import React, {useEffect, useState} from 'react'
 import "./ExternalAccountsPreview.css"
-import MyDataHelps, { ExternalAccount } from "@careevolution/mydatahelps-js"
-import { Action } from '../../presentational'
+import MyDataHelps, {ExternalAccount} from "@careevolution/mydatahelps-js"
+import {Action} from '../../presentational'
 import language from '../../../helpers/language'
-import { previewAccounts } from './ExternalAccountsPreview.previewdata'
+import {previewAccounts} from './ExternalAccountsPreview.previewdata'
 
 export interface ExternalAccountsPreviewProps {
-	excludeDeviceManufacturers?: boolean
-	applicationUrl: ExternalAccountsApplicationUrl,
-	previewState?: ExternalAccountsPreviewPreviewState
+    excludeProviders?: boolean;
+    excludeHealthPlans?: boolean;
+    excludeDeviceManufacturers?: true;
```

Comment:
This looks weird to me?  I would have expected this to also just say `boolean` (because I don't think you can set default values for an interface?) - and also you seem to be checking the truthiness of this below https://github.com/CareEvolution/MyDataHelpsUI/pull/13/files?diff=unified&w=1#diff-c1bad721c4ed87fb7ed25796eb12273532aca486dc0f39e4a124bf4e3d4256e9R36

Reply:
Good catch.  That's a typo.

Reply:
Fixed.